content/03.results.md

```diff
@@ -121,20 +105,14 @@ No additional plots are produced, but a table summarizing the results from all t
 Some samples also include data from bulk RNA-seq and/or spatial transcriptomics libraries.
 Both of these additional sequencing methods are supported by `scpca-nf`.
 To quantify bulk RNA-seq data, `scpca-nf` takes bulk FASTQ files as input, trims reads using `fastp` [@doi:10.1093/bioinformatics/bty560], and then aligns and quantifies reads with `salmon` (Figure {@fig:figS3}A) [@doi:10.1038/nmeth.4197].
-The output is a single TSV file with the gene-by-sample counts matrix for all samples in a given ScPCA project.
-This gene-by-sample matrix is only included with project downloads on the Portal.
+The output is a single TSV file with the gene-by-sample counts matrix for all samples in a given ScPCA project, which is only included with project downloads on the Portal.
```

Comment:
I wonder if we want to save the clause about downloads for teh section where we discuss downloads (which already includes this info)? 
```suggestion
The output is a single TSV file with the gene-by-sample counts matrix for all samples in a given ScPCA project.
```